### PR TITLE
Add bench for copy circuit

### PR DESCRIPTION
**Pull request by smtmfft**
Benchmark test case for copy circuit.
Address the last one of #666

### CODE REVIEW

**Comment by Brechtpd** [comment]
When running with `DEGREE > 14` I get

```
thread 'copy_circuit::tests::bench_copy_circuit_prover' panicked at 'total_rws + 1 <= max_rws, total_rws=14909, max_rws=8192', bus-mapping/src/circuit_input_builder.rs:207:13
```

When running with `DEGREE < 14` I get an `NotEnoughRowsAvailable` error. Is `14` the minimum supported degree of the circuit?

**Comment by smtmfft** [comment]
For degree < 14 maybe yes, I remember it failed even only 1 copy event (will double check). but for degree > 14, no, it is just limited by the CircuitsParams, I forgot to make it grow with the degree. Will fix soon.

**Comment by smtmfft** [comment]
Fixed, now the degree >10 works fine.

**Review by Brechtpd — APPROVED** [review]

**Review by CPerezz — CHANGES_REQUESTED** [review]
1st, thanks for the work on adding benchmarks for the copy circuit.
The doc comments included for the CopyCircuit aren't acceptable.

Please, take just 30 more seconds to put them propperly. Then this work can be merged!

File: zkevm-circuits/src/copy_circuit.rs [hunk 1]
```diff
@@ -586,32 +586,38 @@ pub mod dev {
...
+    /// copy circuit tester
+    pub struct CopyCircuitTester<F> {
         block: Option<Block<F>>,
         max_txs: usize,
         randomness: F,
     }
 
     impl<F: Field> CopyCircuitTester<F> {
-        fn get_randomness() -> F {
+        /// get randomness
```
[the remark this answers is not in the document]
Comment (smtmfft) [reply zkevm-circuits/src/copy_circuit.rs]: Sure. Fixed!

**Review by CPerezz — APPROVED** [review]
I was going to say that the `randomness` should be replaced by the `Challenges`. 
But I see that's already another issue at https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/955.

So let's just approve this!

**Comment by smtmfft** [comment]
Thanks! Not sure if that #955 will be done by copy_circuit writer, if not, we can do that later.

**Pull request merged**